examples/causal_inference/interventional_distribution.ipynb

```diff
@@ -0,0 +1,1561 @@
+{
```

Comment:
<p>Would it be more clear to have $P(x_3 = 1)$ on the RHS and say in text that $P(x_3 = 1) = 1$ in the interventional graph?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I think I see where you are coming from. What I've got matched up with section 1.3 of Pearl's Causality book. I think it makes more sense this way because the truncated factorisation eliminates terms.</p>

Reply:
<p>In a previous edit, I've added more mathematical detail about the factorisation, so hopefully this should be clear to people.</p>

Reply:
I like the new section!! However, I must admit that I tend to prefer more formulas then less, which may or may not be the best for introductory material...